chrome/fed-modules.json

```diff
@@ -0,0 +1,90 @@
+[
+    {
+        "moduleScope": "userPreferences",
+        "manifestLocation": "/apps/user-preferences/fed-mods.json"
```

Comment:
I know we want it to be as permissive as possible, but a lot of these have default values. Might be good to have this only if it differs from `/apps/${moduleScope}/fed-mods.json`

Reply:
We talked about the fed modules files and we will change it so it will be a Routes definition instead. Looking somehing like this:
```json
{
    "dashboard": {
        "manifestLocation": "/apps/dashboard/fed-mods.json",
        "modules": [
            {
                "id": "rhel-dashboard",
                "module": "./RootApp",
                "pathname": [
                    "/insights", 
                    "/insights/dashboard"
                ]
            },
            {
                "id": "ansible-dashboard",
                "module": "./RootApp",
                "pathname": [
                    "/ansible",
                    "/insights/dashboard"
                ]
            }
        ]
    },
}
```

This will allow us to easily identify which module to use and each module will have its own `Route` so there won't be any conflicts between different modules.